Gruntfile.js

```diff
@@ -167,7 +167,30 @@ module.exports = function(grunt) {
 					watchTask: true
```

Comment:
It looks like you are changing something, then undoing the change, then changing it again, not you're undoing it again